fix server invite links

LuanVSO opened the pull request:
add url handler for the discord protocol

kwizart commented:
@LuanVSO thanks for the PR.

You have changed some space indentation, so this makes the commit harder to read.
Can you "amend" the commit to keep the same space indentation and add comment about context to understand why this change is needed ? (and why it's not handled by discord upstream eventually) ?

LuanVSO committed:
add suport for discord url scheme

this will allow the discord website to detect and hand over actions to
the app such as joining a server through a invite link.

these are the same arguments that are used to implement the feature on
windows

implemented as a separate hidden desktop file to keep the main file
without any extra arguments.

this feature request was also send to discord support, so may not be
necessary to do all of this in the future.

LuanVSO commented:
i guess it could be implemented upstream, it would require going through support though and it would like to have an implementation to point to.
should i close this and just send the info to discord support?
